python/cuml/metrics/pairwise_distances.pyx

```diff
@@ -136,6 +134,104 @@ def _determine_metric(metric_str, is_sparse=False):
         return PAIRWISE_DISTANCE_METRICS[metric_str]
 
 
+def nan_euclidean_distances(
```

Comment:
Add `"nan_euclidean"` to `PAIRWISE_DISTANCE_METRICS`

Reply:
Thank you Corey for all the changes suggested will do the relevant changes and commit again.